src/ui/src/predefinedfilterscombobox.cpp

```diff
@@ -102,6 +102,11 @@ PredefinedFiltersComboBox::PredefinedFiltersComboBox( QWidget* parent )
     setSizeAdjustPolicy( QComboBox::AdjustToContents );
 }
 
+PredefinedFiltersComboBox::~PredefinedFiltersComboBox()
+{
+    delete model_;
```

Comment:
Setting this as a parent for the model object in PredefinedFiltersComboBox constructor is a more "Qt" way  to manage this without creating explicit destructors.

Reply:
> Setting this as a parent for the model object in PredefinedFiltersComboBox constructor is a more "Qt" way to manage this without creating explicit destructors.

leave this change?